addon/ng2/commands/test.ts

```diff
@@ -14,7 +14,7 @@ module.exports = TestCommand.extend({
   ],
 
   run: function (commandOptions) {
-    this.project.ngConfig = this.project.ngConfig || config.CliConfig.fromProject();
+    this.project.ngConfig = this.project.ngConfig || CliConfig.fromProject();
```

Comment:
Actually this is correct. We use `ngConfig.config` everywhere where it's needed. I do think `config` is a misnomer and might be better off as `root`, but I have to refactor some usage of this anyway. That might happen later on (with Feenix).